### PR TITLE
Account app insights

### PR DESCRIPTION
**Pull request by dallaspward**
# Add app insights into account, 203

## change proposal

Inform us of the changes that will be applied with this pull request.

- [x] APP_INSIGHTS environment variable inside appsettings.json
- [x] ApplicationInsights.AspNetCore nuget package to revature.account.api.csproj
- [ ] ApplicationInsights.PerfCounterCollector to revature.account.api.csproj
- [ ] Application Insights Telemetry to ConfigureServices in Startup.cs in revature.account.api
- [ ] Health checks to ConfigureServices in Startup.cs in revature.account.api

## reviewer

- @jamesgoldsmith8 
- @fredbelotte 

## contributing

this project is maintained under the terms of the [Contribution Guidelines][contributing].

[contributing]: https://github.com/fredbelotte/alpine/blob/master/.github/CONTRIBUTING.md 'contribution guidelines'

### CODE REVIEW

File: account/src/Revature.Account.Api/appsettings.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+  "APPINSIGHTS_INSTRUMENTATIONKEY": "Putkeyhereplease",
```
Comment (fredbelotte) [inline account/src/Revature.Account.Api/appsettings.json]: follow guidelines

Comment (dallaspward) [reply account/src/Revature.Account.Api/appsettings.json]: Change made

File: account/src/Revature.Account.Api/appsettings.json [hunk 2]
```diff
@@ -1,4 +1,10 @@
 {
+  "APPINSIGHTS_INSTRUMENTATIONKEY": "PutKeyHerePlease",
```
Comment (fredbelotte) [inline account/src/Revature.Account.Api/appsettings.json]: need to fix this.

Comment (dallaspward) [reply account/src/Revature.Account.Api/appsettings.json]: Fixed.

**Pull request merged**